### DISCUSSION TITLE
How to remove "export" buttons

### DISCUSSION THREAD

**Discussion by rockwellwindsor** [upsetjs/upsetjs#125] [General]
I am sure I am just missing it so sorry about the ping here. 

How do you hide the 4 buttons that appear in the bottom right of the Venn Diagram?

All I have used so far is the starter code to begin the build, but I suspect this will be requested and I cannot find a working solution, thanks!

```
const elems = useMemo(
        () => [
          { name: 'A', sets: ['S1', 'S2'] },
          { name: 'B', sets: ['S1'] },
      { name: 'C', sets: ['S2'] },
      { name: 'D', sets: ['S1', 'S3'] },
    ],
    []
  );

  const sets = useMemo(() => extractSets(elems), [elems]);
  const combinations = useMemo(() => generateCombinations(sets), [sets]);

  const [selection, setSelection] = React.useState(null);

  return (
    <VennDiagram
      sets={sets}
      combinations={combinations}
      width={780}
      height={400}
      selection={selection}
      onHover={setSelection}
    />
  );
```

**Comment by rockwellwindsor** [comment]
I did get this solution to work, not sure if there is a JS option to just disable or something.

```
g[class^='exportButtons-upset-'], g[class*='exportButtons-upset-']{
  display:none;
}
```

**Comment by sgratzl** [comment]
```
...
      exportButtons={false}
```
setting the `exportButtons` property to false should do the trick, see https://upset.js.org/api/react/interfaces/VennDiagramStyleProps.html#exportButtons

**Reply by rockwellwindsor** [reply]
Perfect, thanks for the answer and link, appreciate it.  Thanks again!
